composer.json

```diff
@@ -29,10 +29,13 @@
         "ext-dom": "*",
         "ext-filter": "*",
         "ext-json": "*",
+        "container-interop/container-interop": "^1.2",
```

Comment:
Do we need a dependency on `container-interop`, or could perhaps just keep `psr/container`?

Reply:
It's imported by code in `src/` but I'll drop it if you'd like me to?

Reply:
Is it possible to surgically remove it without breaking BC? :thinking:

Reply:
I don't think so, because it's using factory interfaces from service manager that declare it as an argument :( - AFAICT, it can only be dropped when we kill the factories in 3.